README.md

```diff
@@ -43,7 +43,9 @@ To use this template when creating a new repository, follow these steps:
 1. Update other files for your project/product/module (see sections below for guidance).
 1. Test the changes and make required adjustments by executing the dev environment
     - The dev environment is configured to use Keycloak for authentication. This requires you to add an entry to your hosts file to resolve the Keycloak URL to your local machine. Add the following line to your system's `hosts` file:
-      `127.0.0.1    keycloak.local`
+        - macOS/Linux: `127.0.0.1    keycloak.local`
+        - Windows: `keycloak.local    127.0.0.1`
+    - authenticate with docker registry: `docker login harbor.delivery.iqgeo.cloud`
```

Comment:
Thoughts? I was just running through the demo based on these instructions and they may not have authenticated yet.